docs/source/frameworks/onnx.rst

```diff
@@ -0,0 +1,329 @@
+	      init.orthogonal_(self.conv2.weight, init.calculate_gain('relu'))
+	      init.orthogonal_(self.conv3.weight, init.calculate_gain('relu'))
+	      init.orthogonal_(self.conv4.weight)
+
+      torch_model = SuperResolutionNet()
+
+   For this tutorial, we will download some pre-trained weights. Note
+   that this model was not trained fully for good accuracy and is used
+   here for demonstration purposes only.
+
+   .. code-block:: python
+
+      # Load pretrained model weights
+      model_url = 'https://s3.amazonaws.com/pytorch/test_data/export/superres_epoch100-44c6958e.pth'
+
+      # Initialize model with the pretrained weights
+      map_location = lambda storage, loc: storage
+      if torch.cuda.is_available():
+	  map_location = None
+      torch_model.load_state_dict(model_zoo.load_url(model_url, map_location=map_location))
+
+      # set the model to inference mode
+      torch_model.eval()
+
+
+   Exporting a model to onnx in PyTorch works via tracing or
+   scripting. In this tutorial we will export a model using
+   tracing. Note how we export the model with an input of
+   ``batch_size=1``, but then specify the first dimension as dynamic
+   in the ``dynamic_axes`` parameter in ``torch.onnx.export()``. The
+   exported model will thus accept inputs of size ``[batch_size, 1,
+   224, 224]`` where ``batch_size`` can vary among each inference.
+
+   .. code-block:: python
+
+      batch_size = 1 # can be any number
+      # Tracing input to the model
+      x = torch.randn(batch_size, 1, 224, 224, requires_grad=True)
+
+      # Export the model
+      torch.onnx.export(torch_model,
+			x,
+			"super_resolution.onnx",   # where to save the model (can be a file or file-like object)
+			export_params=True,        # store the trained parameter weights inside the model file
+			opset_version=10,          # the ONNX version to export the model to
+			do_constant_folding=True,  # whether to execute constant folding for optimization
+			input_names = ['input'],   # the model's input names
+			output_names = ['output'], # the model's output names
+			dynamic_axes={'input' : {0 : 'batch_size'},    # variable length axes
+				      'output' : {0 : 'batch_size'}})
+
+   Now we can compute the output using ONNX Runtime’s Python APIs:
+
+   .. code-block:: python
+
+      import onnxruntime
+
+      ort_session = onnxruntime.InferenceSession("super_resolution.onnx")
+      # compute ONNX Runtime output prediction
+      ort_inputs = {ort_session.get_inputs()[0].name: to_numpy(x)}
+      # ONNX Runtime will return a list of outputs
+      ort_outs = ort_session.run(None, ort_inputs)
+      print(ort_outs[0])
+
+.. tab:: TensorFlow
+
+   TODO
+
+.. tab:: Scikit Learn
+
+   TODO
+
+
+Saving ONNX model with BentoML
+-----------------------------
+
+To quickly save an ONNX model to BentoML's :ref:`Model
+Store<concepts/model:Managing Models>`, first use ``onnx.load`` to
+load the exported ONNX model back into ``onnx.ModelProto`` object,
```

Comment:
would be nice if you can reference the `onnx.ModelProto` to the API documentation.

Reply:
I feel like save model should be a small section under the first block `Train a model and export to ONNX format` and no need to get into how batchable is disabled by default. The general tutorial should already covered that.

Reply:
I still think for framework support batching, we should talk about `batchable` because the default `batchable=False` may not result optimized performance @aarnphm

Reply:
noted.